runtime/doc/eval.txt

```diff
@@ -2041,6 +2041,7 @@ garbagecollect([{atexit}])	none	free memory, breaking cyclic references
 get({list}, {idx} [, {def}])	any	get item {idx} from {list} or {def}
 get({dict}, {key} [, {def}])	any	get item {key} from {dict} or {def}
 get({func}, {what})		any	get property of funcref/partial {func}
+get_xdg_home({string})          String  get the XDG Base directory for {string}
```

Comment:
XDG _does_ apply to Windows.  See [`stdpaths.c`](https://github.com/neovim/neovim/blob/2a366b674aa572e7cbdef257c5597cf62c9a6f5f/src/nvim/os/stdpaths.c#L30-L49).

Though now that I think of it, it should probably be something closer to the `stdpaths_user_conf_subpath()` and it's `_data_` sibling since it outputs the `nvim-data` directory for Windows.

I don't like `config_path()` because there is a data and cache directory too.